src/core/components/layout/Hide/Hide.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import { Hide, HideProps } from './Hide';
+import { asChromaticStory, asPlayground } from '../../../../lib/story-intents';
+import { Story } from '../../../../@types/storybook-emotion-10-fixes';
+
+export default {
+	title: 'Source/src-layout/Hide',
+	component: Hide,
+	argTypes: {
+		until: {}, // display first
+		from: {}, // display first
```

Comment:
Did we decide we're just going to get rid of these for now then?

Reply:
nice one